src/App.js

```diff
@@ -0,0 +1,13 @@
+import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
+}
```

Comment:
- [ ] The `App.js` component should be class-based. Please follow the microverse guidelines and also it's better to learn class-based component and to apply it in your code at the same time.

Reply:
_Why App.js component should be class-based?_

I already applied a class-based component inside Calculator.js

Reply:
You have to apply the same in `App.js` as well since it's mentioned in the microverse guideline and also it's better to apply in all places so that you'll get an idea to use class based on a different type of component.

![Screenshot from 2021-08-23 17-34-09](https://user-images.githubusercontent.com/22434660/130444082-0ac0641c-3a52-4bc0-b4b3-8a0a94d7edc1.png)